cmd/daily.go

```diff
@@ -12,26 +12,27 @@ import (
 var dailyPeriod string
 var dailyPkg string
 
-var dailyCmd = &cobra.Command{
-	Use:   "daily",
-	Short: "Get daily downloads of a package",
-	Run: func(cmd *cobra.Command, args []string) {
-		daily, err := data.GetDaily(dailyPeriod, dailyPkg)
+func daily(_ *cobra.Command, args []string) {
+	daily, err := data.GetDaily(dailyPeriod, dailyPkg)
 
-		if err != nil {
-			color.PrintError(err.Error())
-			os.Exit(0)
-		}
+	if err != nil {
+		color.PrintError(err.Error())
+		os.Exit(0)
```

Comment:
@dpastoor do we need to to exit here? Can't we just skip to the next package with `continue`?

Reply:
yeah - I left that be for the moment, as kinda mentioned in the comment, wanted to get your preference first. IMO, I'd rather capture all the errors, then spit them out at the end, so I wouldn't exit. "give the user as much information as possible when possible" to hopefully minimize iteratively working through each error

I'll drop a PR with how I'd do it, though if you have alternative designs I'm happy to implement it however